tests/scripts/thread-cert/Cert_5_5_07_SplitMergeThreeWay.py

```diff
@@ -111,6 +117,86 @@ def test(self):
+        router3_pkts = pkts.filter_wpan_src64(ROUTER_3)
+
+        leader_pkts.filter_mle_cmd(MLE_CHILD_ID_RESPONSE).must_next()
+        _start_idx = router3_pkts.index
+        _lpkts = leader_pkts.range(_start_idx)
+        _router1_pkts = router1_pkts.range(_start_idx)
+
+        # Step 2: The Leader and Router_1 MUST send properly formatted MLE Advertisements
+        leader_pkts.filter_mle_cmd(MLE_CHILD_ID_RESPONSE).must_next()
+        _lpkts = leader_pkts.range(leader_pkts.index)
+        p = _lpkts.filter_mle_cmd(MLE_ADVERTISEMENT).must_next()
+        self.assertTrue([0, 11, 9] == p.mle.tlv.type)
+
+        _router1_pkts = router1_pkts.range(leader_pkts.index)
+        p = _router1_pkts.filter_mle_cmd(MLE_ADVERTISEMENT).must_next()
+        self.assertTrue([0, 11, 9] == p.mle.tlv.type)
+
+        # Step4: Each router forms a partition with the
+        # lowest possible partition ID
+        # Step 5: Router_1 MUST send MLE Parent Requests
+        # and MUST make two separate attempts
+        p = _router1_pkts.filter_mle_cmd(MLE_PARENT_REQUEST).must_next()
+        self.assertTrue([1, 3, 14, 18] == p.mle.tlv.type)
+        self.assertEqual(p.mle.tlv.scan_mask.r, 1)
+        self.assertEqual(p.mle.tlv.scan_mask.e, 1)
+        lreset_start = _router1_pkts.index
+        p = _router1_pkts.filter_mle_cmd(MLE_PARENT_REQUEST).must_next()
+        self.assertTrue([1, 3, 14, 18] == p.mle.tlv.type)
+        self.assertEqual(p.mle.tlv.scan_mask.r, 1)
+        self.assertEqual(p.mle.tlv.scan_mask.e, 1)
+
+        # Step 7:Router_1 MUST attempt to attach to any other Partition
+        # within range by sending a MLE Parent Request.
+        p = _router1_pkts.filter_mle_cmd(MLE_PARENT_REQUEST).must_next()
+        lreset_stop = _router1_pkts.index
+        self.assertTrue([1, 3, 14, 18] == p.mle.tlv.type)
+
+        # Step 6: The Leader does not respond to the Parent Requests
+        p = leader_pkts.range(lreset_start, lreset_stop)
+        p.filter_mle_cmd(MLE_PARENT_RESPONSE).must_not_next()
+
+        # Step3: The Leader MUST stop sending MLE advertisements.
+        p = leader_pkts.range(lreset_start, lreset_stop)
+        p.filter_mle_cmd(MLE_ADVERTISEMENT).must_not_next()
+
+        # Step 8: Router_1 take over leader role of a new Partition and
+        # begin transmitting MLE Advertisements
+        p = _router1_pkts.copy().filter_mle_cmd(MLE_ADVERTISEMENT).must_next()
+        self.assertTrue([0, 11, 9] == p.mle.tlv.type)
+
+        # Step 9: The Leader MUST send properly formatted MLE Parent
+        # Requests to the All-Routers multicast address
+        p = _lpkts.filter_mle_cmd(MLE_PARENT_REQUEST).must_next()
+        self.assertTrue([1, 3, 14, 18] == p.mle.tlv.type)
+
+        # Step 10: Router_1 MUST send an MLE Parent Response
+        p = _router1_pkts.filter_mle_cmd(MLE_PARENT_RESPONSE).must_next()
+        self.assertTrue(set([0, 11, 5, 4, 3, 16, 15, 18]) < set(p.mle.tlv.type))
```

Comment:
```suggestion
        self.assertTrue({0, 11, 5, 4, 3, 16, 15, 18} <= set(p.mle.tlv.type))
```
btw, use `must_verify`